bcda/cli.go

```diff
@@ -211,3 +211,29 @@ func sortCCLFFiles(cclf0, cclf8, cclf9 *[]cclfFileMetadata, skipped *int) filepa
 		return nil
 	}
 }
+
+func deleteDirectory(dirToDelete string) (filesDeleted int, err error) {
+	log.Info(fmt.Sprintf("preparing to delete directory '%v'", dirToDelete))
```

Comment:
Technically, we aren't deleting the directory, right?  We are just deleting the files in the directory.

Reply:
Renamed function to be more clear.